### DISCUSSION TITLE
Circular reference to CLI11 directory

### DISCUSSION THREAD

**Discussion by InscrutableJack** [sergeyklay/pcloud-console-client#98] [Q&A]
Opting for discussion rather than an issue as this is likely use-case related.

I'm trying to setup the project as a remote build in CLion, but pcloud-console-client\external\CLI11\tests\mesonTest\subprojects\CL11 links back to the parent CLI11 in the path.  When CLion walks the directories to load the CMake project, it infinitely recurses.

I may be able to ignore that link, but I wanted to confirm it's intentionally there, or if not, call your attention to it.

**Comment by sergeyklay** [comment]
Hmm, interesting... Can you provide a bit more detailed steps to reproduce?

**Reply by InscrutableJack** [reply]
Sorry for the delay; I've had a lot of other things going on.

Basically you open the project from the top-level CMake file, and assign a functional remote build toolchain to it (for example, a Linux machine with cmake and gcc build tools and SSH access).  CLion will start loading the project again, and if you click on the progress bar at the bottom of the window to expand the individual processes, you will see the file paths being processed.  Once it reaches the CLI11 shortcut, it recurses back and starts re-walking its subdirectories indefinitely.
